src/arrays/ladderops.jl

```diff
@@ -16,11 +16,11 @@
     lowermatrix(lens, particle) = eye_sandwich(lens, particle, lower_single(lens[particle]))
 
     laddercoeffs(n) = sqrt(linspace(1, n, n))
-    lower_single(n) = sparse([1:n-1], [2:n], laddercoeffs(n-1), n, n)
-    raise_single(n) = sparse([2:n], [1:n-1], laddercoeffs(n-1), n, n)
+    lower_single(n) = sparse([1:n-1;], [2:n;], laddercoeffs(n-1), n, n)
+    raise_single(n) = sparse([2:n;], [1:n-1;], laddercoeffs(n-1), n, n)
```

Comment:
Oh, nice! I didn't know about the `name` field. In that case, yeah, `typerepr` can be removed. If you want to remove it now before merging you can, otherwise I'll do so later.